src/project.mk

```diff
@@ -78,7 +78,7 @@ ifneq ($(REALM_ANDROID),)
   PROJECT_CFLAGS += -fPIC -DPIC
   # android.toolchain.cmake has `-fsigned-char` by default, we have to use the same
   # to avoid lto linking problems.
-  CFLAGS_OPTIM = -Os -flto -ffunction-sections -fdata-sections -DNDEBUG -fsigned-char -fvisibility=hidden
+  CFLAGS_OPTIM = -Os -DNDEBUG -fsigned-char -fvisibility=hidden
```

Comment:
This change also removes some flags related to dead code (and data) elimination?